src/elli.erl

```diff
@@ -171,14 +176,14 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 
 remove_acceptor(State, Pid) ->
-    State#state{acceptors = lists:delete(Pid, State#state.acceptors),
```

Comment:
If someone upgrades to this version of elli on a running system it will crash on the first start/stop of an acceptor process, right?